.github/workflows/comment_bot.yml

```diff
@@ -53,5 +53,72 @@ jobs:
           archery trigger-bot \
             --event-name ${{ github.event_name }} \
             --event-payload ${{ github.event_path }}
-
-
+  autotune:
+    name: "Fix all the things"
+    if: startsWith(github.event.comment.body, 'autotune')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: r-lib/actions/pr-fetch@master
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+      - name: See what is different
+        run: |
+          set -ex
+          git remote add upstream https://github.com/apache/arrow
+          git fetch upstream
+          changed() {
+            git diff --name-only HEAD^..upstream/master | grep -e "$1" >/dev/null 2>&1
+          }
+          if changed '^r/.*\.R$'; then
+            echo "::set-env name=R_DOCS::true"
+          fi
+          if changed 'cmake' || changed 'CMake'; then
+            echo "::set-env name=CMAKE_FORMAT::true"
+          fi
+          if changed '^cpp/src'; then
+            echo "::set-env name=CLANG_FORMAT_CPP::true"
+          fi
+          if changed '^r/src'; then
+            echo "::set-env name=CLANG_FORMAT_R::true"
+          fi
+      - name: Run cmake_format
+        if: false
+        # if: env.CMAKE_FORMAT == 'true' || endsWith(github.event.comment.body, 'everything')
```

Comment:
Debug code?

Reply:
Per the description: 

> * cmake_format is currently disabled because I didn't feel like fighting further with python versions and environments; in https://github.com/nealrichardson/arrow/runs/590355718?check_suite_focus=true#step:5:34 you can see that it pip installs cmake_format but then in the next line can't find cmake_format 🤷

Reply:
Oops, sorry I missed that. Could you add a follow up giraffe and a matching to do comment?

Reply:
🦒 done in https://github.com/apache/arrow/pull/6932/commits/5bfce2dc160b65d111d49114c8660b1f1a498e19